examples/lighting-app/nrfconnect/main/include/Rpc.h

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <kernel.h>
+
+#ifdef CONFIG_CHIP_PW_RPC
+#pragma message "Building with RPC support"
+#include "pw_sys_io_nrfconnect/init.h"
+
+namespace {
+#define RPC_STACK_SIZE 4096
+#define RPC_PRIORITY 5
+
+K_THREAD_STACK_DEFINE(rpc_stack_area, RPC_STACK_SIZE);
```

Comment:
Is it really OK to define this in a header? If the header is included more than once, we get multiple distinct definitions.

Reply:
This is a better argument for moving the #ifdef and definitions to Rpc.cpp

Reply:
done